src/gql/generated/types.ts

```diff
@@ -1502,7 +1502,7 @@ export type ProjectEventSettings = {
 
 export type RepoEventSettings = {
   githubWebhooksEnabled: Scalars["Boolean"];
-  projectRef?: Maybe<Project>;
+  projectRef?: Maybe<RepoRef>;
```

Comment:
This app change is a breaking change, but since the event page isn't used anywhere or tested yet, I think there should be no impact 👍